{
  "config": {
      "abort": {
          "already_configured": "Device is already configured"
      },
      "error": {
          "cannot_connect": "Failed to connect",
          "invalid_auth": "Invalid authentication",
          "unknown": "Unexpected error"
      },
      "step": {
          "user": {
              "data": {
                  "host": "Host",
                  "password": "Password",
                  "scan_interval": "Seconds between updates",
                  "username": "Username"
              },
              "description": "Set connexion settings."
          }
      }
  },
  "options": {
      "error": {
          "cannot_connect": "Failed to connect",
          "invalid_auth": "Invalid authentication",
          "unknown": "Unexpected error"
      },
      "step": {
          "init": {
              "data": {
                  "host": "Host",
                  "password": "Password",
                  "scan_interval": "Seconds between updates",
                  "username": "Username"
              },
              "description": "Update settings"
          }
      }
  }
}